src/cmd_argument.rs

```diff
@@ -37,7 +37,7 @@ impl CmdArgument for String {
 }
 
 /// See the [`CmdArgument`] implementation for [`Split`] below.
-pub struct Split<'a>(pub &'a str);
+pub struct Split<T: AsRef<str>>(pub T);
```

Comment:
Why not use `Into<String>`?

Reply:
If the argument is of type `&str`, `Into<String>` would be a clone, which isn't necessary, since `split_whitespace` takes `&self`.

Reply:
Right, thanks for explaining.